src/index.css

```diff
@@ -1,5 +1,64 @@
+@font-face {
```

Comment:
Some of sytles should be in a `<component>.styles.js` don't you think ? (with glamor)

Reply:
I moved the specific styles in style files of the concerned component.
Here are only styles shared by 2 components